### PR TITLE
Add-proxy-to-subscription-if-notebooks

### PR DESCRIPTION
**Pull request by pgladkows**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -190,7 +190,15 @@ ifeq ($(TARGET_PLATFORM), openshift)
 else
 	kubectl create ns operator || true
 	kubectl apply -f tests/operator-group.yaml
+ifeq ($(ADD_NOTEBOOK_K8S), 1)
+	sed -i "s|<http_proxy>|$(http_proxy)|" tests/operator-subscription-proxy.yaml
+	sed -i "s|<https_proxy>|$(https_proxy)|" tests/operator-subscription-proxy.yaml
+	sed -i "s|<no_proxy>|$(no_proxy)|" tests/operator-subscription-proxy.yaml
```
Comment (mzegla) [inline Makefile:196]: What happens if proxy envs are not set?

**Review by dtrawins — APPROVED** [review]

File: tests/operator-subscription.yaml [hunk 2]
```diff
@@ -4,6 +4,14 @@ metadata:
...
+    env:
+    - name: HTTP_PROXY
+      value: <http_proxy>
+    - name: HTTPS_PROXY
+      value: <https_proxy>
+    - name: NO_PROXY
+      value: <no_proxy>
   channel: alpha
   name: ovms-operator
   startingCSV: openvino-operator.v1.0.0
```
Comment (mzegla) [inline tests/operator-subscription.yaml:17]: Could you also update this line to v1.1.0 ?

Comment (pgladkows) [reply tests/operator-subscription.yaml]: Done.

**Review by mzegla — APPROVED** [review]

File: Makefile [hunk 3]
```diff
@@ -190,6 +190,21 @@ ifeq ($(TARGET_PLATFORM), openshift)
 else
 	kubectl create ns operator || true
 	kubectl apply -f tests/operator-group.yaml
+ifneq ($(http_proxy),)
```
Comment (dtrawins) [inline Makefile:193]: can we avoid those if conditions? maybe with the "<http_proxy>" in quotes?

Comment (pgladkows) [reply Makefile]: Good point, I corrected that.

**Review by dtrawins — APPROVED** [review]

**Pull request merged**